### PR TITLE
Fix banner link 

### PR DESCRIPTION
**Pull request by vitoriaHeliane**
#### What is the purpose of this pull request?
Fix banner link.

#### What problem is this solving?
Banner internal and external links weren't working.

#### How should this be manually tested?
[Access the workspace](https://carouselfix--storecomponents.myvtex.com/).

#### Types of changes
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 ### Changed
 - **slick slider** Integrate with vtex.storecomponents/Slider
+### Fixed
+- Fix banner link working.
```
Comment (andreldsa) [inline CHANGELOG.md]: Turn that comment better.

Comment (ClaudivanFilho) [inline CHANGELOG.md]: Fix bug of banner link not working

File: react/Banner.js [hunk 2]
```diff
@@ -14,22 +27,30 @@ class Banner extends Component {
       page,
       params,
       mobileHeight,
+      typeOfRoute,
     } = this.props
 
+    const content = (
+      <div className="img-container">
+        <img className="img-regular w-100" src={image} alt={description} />
```
Comment (ClaudivanFilho) [inline react/Banner.js]: prefix vtex-

File: react/Carousel.js [hunk 3]
```diff
@@ -123,20 +123,20 @@ export default class Carousel extends Component {
       typeOfRoute === 'external' ? {
```
Comment (brunojdo) [inline react/Carousel.js]: To maintain the consistency, you should invert to internal first, like you do on `<Link />`

File: react/Carousel.js [hunk 4]
```diff
@@ -123,20 +123,20 @@ export default class Carousel extends Component {
...
         page: {
           type: 'string',
-          title: 'Banner link',
+          title: 'Banner link (should start with https)',
```
Comment (ClaudivanFilho) [inline react/Carousel.js]: I think that is better to remove this `(should start with https)` and put a default value `https://`

File: react/Banner.js [hunk 5]
```diff
@@ -14,22 +27,30 @@ class Banner extends Component {
...
-              <img className="w-100" src={mobileImage} alt={description} />
-            </div>
-          </div>
+      typeOfRoute === 'internal' ?
+        <Link page={page} params={this.getParams(params)}>
+          {content}
         </Link>
-      </div>
+        :
+        <a href={page} target="_blank">
```
Comment (andreldsa) [inline react/Banner.js]: You encapsulated the content div inside the anchor tag, destructuring the CSS styling.

File: CHANGELOG.md (hunk 1 above)
Comment (brunojdo) [reply CHANGELOG.md]: Fix issue that makes banner link not working properly 💃

**Pull request merged**